sources/api/apiclient/README.md

```diff
@@ -19,40 +52,41 @@ Specify the data after `-d` or `--data`.
 
 To see verbose response data, including the HTTP status code, use `-v` or `--verbose`.
 
-### Example usage
+#### Examples
 
 Getting settings:
 
 ```
-apiclient -m GET -u /settings
+apiclient raw -m GET -u /settings
 ```
 
 Changing settings:
 
 ```
-apiclient -X PATCH -u /settings -d '{"motd": "my own value!"}'
-apiclient -m POST -u /tx/commit_and_apply
+apiclient raw -X PATCH -u /settings -d '{"motd": "my own value!"}'
```

Comment:
Was thinking if this would break existing automation of customer who might be using older api's in their script without `raw` prefix.

Reply:
The old way of using `apiclient` is still supported. The `raw` subcommand is the default command and can be omitted: https://github.com/bottlerocket-os/bottlerocket/blob/ccc864a905607ca4a7cd349a5679826b3b9f58e4/sources/api/apiclient/src/main.rs#L87-L89

Reply:
I was just trying to encourage the new syntax, but yeah, the old syntax is still fine.